README.md

```diff
@@ -48,3 +48,8 @@ If we can hit that bullseye, the rest of the dominoes will fall like a house of
 
 As of July 2019, Java 8 is supported.
 Support for Java 6 and Java 7 has ended.
+
+## Release process
+When a PR is merged to master, a release will be made and deployed to Sonatype by Github Actions. This includes pushing new commits via `maven-release-plugin`. The released version will be automatically deployed and usable immediately after the `release` workflow completes.
+
+To use the manual release process, see https://renttherunway.jira.com/wiki/spaces/EN/pages/100991186/How+to+Release+Conduit
```

Comment:
are you sure we want to have this wiki link in this repo?

Reply:
I'll update the page to indicate that it's not the preferred process anymore but it is still the fall-back so probably useful to include.